packages/account/src/Components/currency-selector/__tests__/currency-selector.spec.tsx

```diff
@@ -269,20 +269,20 @@ describe('<CurrencySelector/>', () => {
                     currency: 'USD',
                     is_disabled: 0,
                     is_virtual: 1,
-                    landing_company_shortcode: 'svg',
+                    landing_company_shortcode: Jurisdiction.SVG,
                     trading: {},
                     token: 'a1-sLGGrhfYPkeEprxEop2T591cLKbuN',
                     email: 'test+qw@deriv.com',
                     session_start: 1651059038,
                     excluded_until: '',
-                    landing_company_name: 'svg',
+                    landing_company_name: Jurisdiction.SVG,
```

Comment:
@suisin-deriv we dont need change anything other than jurisdiction part.. These landing company anf real account sighnup is not part of juridsiction

Reply:
owhh okok understand, so I will only change those which is related to Jurisdiction modal only?